### PR TITLE
Add sterling module

### PR DESCRIPTION
**Pull request by mezlet**

### CODE REVIEW

File: lib/sterling/common/Request.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const request = require('../../common/Request');
+const { Handler, ApiErrors, ModuleError } = require('../../common/utils/errors');
+
+exports.trigger = async({
+    method = 'POST', path, credentials, payload,
+}) => {
+    const baseURL = credentials.host || 'https://sandboxapi.fsi.ng';
```
Comment (akinjide) [inline lib/sterling/common/Request.js]: Let's move baseURL, common request options to base `Request.http`

Comment (mezlet) [reply lib/sterling/common/Request.js]: noted

File: lib/nibss/common/Request.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+const { Handler, ApiErrors, ModuleError } = require('../../common/utils/errors');
+
+exports.trigger = async({
+    method = 'POST', path, credentials, payload,
+}) => {
+    let headers,
+        encryptedBVN,
+        bvnData;
+
+    const { host } = credentials;
```
Comment (akinjide) [inline lib/nibss/common/Request.js]: Why have all this outside try block and conditional check?

File: lib/nibss/common/Request.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            host,
+            path,
+            options,
+        });
+        if (response.statusCode !== 200) {
+            return new ApiErrors(Handler.Api(response.statusCode));
+        }
+        if (response.body) {
+            data = JSON.parse(hash.decrypt(response.body, credentials.aes_key, credentials.ivkey));
+            return data;
```
Comment (akinjide) [inline lib/nibss/common/Request.js]: no reason to assign data just return

File: lib/nibss/common/Request.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        }
+
+        data = {
+            password: response.headers.password,
+            ivkey: response.headers.ivkey,
+            aes_key: response.headers.aes_key,
+        };
+        return data;
```
Comment (akinjide) [inline lib/nibss/common/Request.js]: unnecessary declaration just `return { password, ivkey, aes_key }`

File: test/fixtures/Bvnr.js [hunk 5]
```diff
@@ -1,6 +1,5 @@
 const faker = require('faker');
 const utf8 = require('utf8');
```
Comment (akinjide) [inline test/fixtures/Bvnr.js]: Is this utf8 lib necessary?

File: test/fixtures/Enquiry.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    Referenceid: "01",
+    RequestType: "01",
+    Translocation: "01",
+    ToAccount: "0037514056",
+    destinationbankcode: "000001"
+   },
+   subscription_key: "t",
+   Appid: 69,
+   ipval: 0
+}
```
Comment (akinjide) [inline test/fixtures/Enquiry.js]: Use random mock data and not actual data

File: lib/nibss/common/Request.js (hunk 2 above)
Comment (mezlet) [reply lib/nibss/common/Request.js]: noted

File: lib/nibss/common/Request.js (hunk 3 above)
Comment (mezlet) [reply lib/nibss/common/Request.js]: noted

File: test/fixtures/Bvnr.js (hunk 5 above)
Comment (mezlet) [reply test/fixtures/Bvnr.js]: noted

File: lib/nibss/common/Request.js (hunk 4 above)
Comment (mezlet) [reply lib/nibss/common/Request.js]: noted

File: test/fixtures/Enquiry.js (hunk 6 above)
Comment (mezlet) [reply test/fixtures/Enquiry.js]: noted

**Pull request merged**